.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -26,35 +25,23 @@ body:
     validations:
       required: true
   - type: textarea
-    id: screenshots
+    id: actual-result
     attributes:
-      label: Screenshots
-      description: Please add screenshots if applicable
+      label: Actual Results
+      description: Add your observations/output/log/screenshots if applicable
     validations:
       required: false
   - type: textarea
-    id: extrainfo
+    id: expected-result
     attributes:
-      label: Additional information
-      description: Is there anything else we should know about this bug? (like linux distribution(version + arch), python version and etc)   
+      label: Expected Results
+      description: what is your expected result?
     validations:
       required: false
```

Comment:
And, if we require the "actual results", I'd like to see this field required, as well.  (Hopefully the user's expectations will be obvious, but I think it's worthwhile to have them state them, just so there's no misunderstanding.)

Reply:
Fixed